extension/src/popup/views/AccountHistory/index.tsx

```diff
@@ -88,18 +89,20 @@ export const AccountHistory = () => {
   const stellarExpertUrl = getStellarExpertUrl(networkDetails);
 
   // differentiate between if data is still loading and if no account history results came back from Horizon
+  const shouldLoadToken =
+    isExperimentalModeEnabled || networkDetails.network === NETWORKS.FUTURENET;
```

Comment:
should this be `&&`? It'll be using `stellar-sdk` instead of `soroban-client` if `isExperimentalModeEnabled` is false